### DISCUSSION TITLE
Problem when i want to flash new firmware with make cload

### DISCUSSION THREAD

**Discussion by ClementSrs** [bitcraze/discussions#1050] [Q&A]
Hi ! 
I want to recover ai deck's images on a specific access point because i receive vicon's data on it. 
So, I configure the $make menuconfig in my crazyflie-firmware folder and I configure the new access point.
Then I build my firmware with $make and then I want to transfer this firmware to my CF via the crazyradio with $make cload.

clement@clement-Latitude-5500:~/crazyflie-firmware$ make
make[1] : on entre dans le répertoire « /home/clement/crazyflie-firmware/build »
  Using .. as source for kernel
  GEN     ./Makefile
python3 ../tools/make/versionTemplate.py --crazyflie-base .. ../src/utils/src/version.vtpl src/utils/src/version_gen.c
  CC      src/utils/src/version_gen.o
  LD      src/utils/src/version.o
  LD      src/utils/src/built-in.o
  LD      src/utils/built-in.o
  LD      src/built-in.o
  LD      firmware
  COPY  cf2.hex
  COPY  cf2.bin
Build for the cf2 platform!
Build 0:dbb9df1137f1 (2023.10) CLEAN
Version extracted from git
Flash |  268304/1032192 (26%),  763888 free | text: 261332, data: 6972, ccmdata: 0
RAM   |   84048/131072  (64%),   47024 free | bss: 77076, data: 6972
CCM   |   62024/65536   (95%),    3512 free | ccmbss: 62024, ccmdata: 0
make[1] : on quitte le répertoire « /home/clement/crazyflie-firmware/build »

The following error appears : 

clement@clement-Latitude-5500:~/crazyflie-firmware/build$ make cload
tools/kbuild/Makefile.kbuild:147: avertissement : surchargement de la recette pour la cible « cload »
Makefile:136: avertissement : ancienne recette ignorée pour la cible « cload »
python3 -m cfloader  flash  cf2.bin stm32-fw
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/clement/.local/lib/python3.8/site-packages/cfloader/__main__.py", line 1, in <module>
    from cfloader import main
ImportError: cannot import name 'main' from 'cfloader' (/home/clement/.local/lib/python3.8/site-packages/cfloader/__init__.py)
make[2]: *** [/home/clement/crazyflie-firmware/Makefile:136 : cload] Erreur 1
make[1]: *** [tools/kbuild/Makefile.kbuild:150 : sub-make] Erreur 2
make: *** [Makefile:24 : __sub-make] Erreur 2

I give you the __main.py__ and the __init.py__ of the cfloader package in python3.8 : 

__init.py__ : 

__author__ = """Shachi Bista"""
__email__ = "shachibista@gmail.com"
__version__ = '0.1.4'

from cfloader.fs import open
from cfloader.readers import Path, Archive

__main.py__ : 

from cfloader import main

if __name__ == "__main__":
    main()

Thank you for your answer

**Comment by tobbeanton** [comment]
That is a strange error. Perhaps something has gone wrong with the installation. Can you try to re-install cfclient?

**Reply by ClementSrs** [reply]
This works TY. 
Clément
